tests/particle_interpolator_quadratic_least_squares_2d_add_particles.prm

```diff
@@ -0,0 +1,127 @@
+# A test based on the bilinear SolKz benchmark that ensures that additional particles
+# are added to the model during the computation with multiple properties on 
+# the particles. 
```

Comment:
Please update this text to say that this test is based on the bilinear SolKz benchmark, but uses the quadratic least squares interpolator.

Reply:
What do you mean by the **bilinear** SolKz benchmark?  As far as I know, SolKz is just a set of initial conditions that produce a solution to steady Stokes flow.  Would it referred to as the **bilinear SolKz benchmark** if it were used as a test for the compositional fields, or particle averages?

Reply:
Oh I was just quoting the current description in that part. I think what "bilinear SolKz benchmark" means is "the SolKz benchmark using particles that use the bilinear least squares interpolation". Feel free to clarify that in the test description.